scala.wake

```diff
@@ -148,10 +148,11 @@ global def runIvyDep dep = resolveIvyDeps (dep, Nil) | runJava
 # - Add Java options (eg. -Xmx4G)
 # visible excludes classpath
 global def runJava classpath main args visible =
-  def java = which "java"
```

Comment:
On this note, I think we probably want to avoid using the `which` keyword in general, especially for "library code", since it inherently escapes the sandbox by resolving the path outside of the Plan/Job/Runner system.